src/components/card/card.scss

```diff
@@ -125,6 +132,11 @@ $_focus-overflow: (size.$edge-large * -1);
         'cover footer footer'
         'cover . .';
       text-align: start;
+
+      .c-card__cover {
+        block-size: auto; /* 1 */
+        inline-size: auto; /* 1 */
```

Comment:
If I remove `inline-size: auto`, it seems to still work fine? Any reason we need both?

Reply:
Also, can we move this style down to the `.c-card__cover` section instead of being a child of the modifier class? That way all of the cover styles are in one place.

Reply:
> If I remove `inline-size: auto`, it seems to still work fine? Any reason we need both?

Good catch, both are likely not needed. 👍🏽

Reply:
> Also, can we move this style down to the `.c-card__cover` section instead of being a child of the modifier class? That way all of the cover styles are in one place.

@tylersticka I'm not sure if I've painted myself into a corner, but the `block-size: auto` needs to be applied as a child of the `.c-card--circle-cover` modifier:

- `.c-card__cover`: Doesn't need explicit `block-size`
- `.c-card--horizontal .c-card__cover`: Needs `block-size` and `inline-size` of `100%`
- `.c-card--horizontal.c-card--circle-cover .c-card__cover`: Needs `block-size: auto` (fixes the bug I noticed)

Have I made it too complex? 🔍 👀

Reply:
Ah, got it. Thanks, @tylersticka, appreciate it!